Calamity/Buffs/ArcaneShock.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Terraria;
+using Terraria.ModLoader;
+
+namespace MasomodeDLC.Calamity.Buffs
+{
+    public class ArcaneShock : ModBuff
+    {
+        public override void SetDefaults()
```

Comment:
You don't need to add the placeholder icon. Look at the other buffs we used. Hook into autoload.

Reply:
Buff icon removed.

Reply:
You didn't hook autoload still.

Reply:
Refer to https://github.com/kittenchilly/MasomodeDLC/blob/master/Thorium/Buffs/teslasurge.cs#L10